pkg/apis/allocation/v1/gameserverallocation.go

```diff
@@ -122,7 +131,26 @@ type PlayerSelector struct {
 	MaxAvailable int64 `json:"maxAvailable,omitempty"`
 }
 
-// ApplyDefaults applies default values to the PlayerSelector
+// CounterSelector is the filter options for a GameServer based on the count and/or available capacity.
+// Undefined or 0 for MaxCount or MaxAvailable means unlimited maximum.
+type CounterSelector struct {
+	MinCount     int64 `json:"minCount"`
```

Comment:
"Count" makes sense to me since it corresponds to the GamerServerStatus CounterStatus.Count. I wouldn't use Value here to represent an integer since we use Value in ListSelector to denote a string in a list of strings (that corresponds to the GamerServerStatus ListStatus.Values). If you prefer less redundancy in naming should I change it to min?